exercises/isbn-verifier/isbn_verifier_test.go

```diff
@@ -10,7 +10,8 @@ func TestIsValidISBN(t *testing.T) {
 		if observed != test.expected {
 			t.Errorf("FAIL: %s\nIsValidISBN(%q)\nExpected: %t, Actual: %t",
 				test.description, test.isbn, test.expected, observed)
+		} else {
+			t.Logf("PASS: %s", test.description)
```

Comment:
Hm, I'm not sure I like the double negative with the "if not / else". Could we switch it around?

Reply:
Of course, i'll do it right away :)